.github/workflows/pylint.yml

```diff
@@ -0,0 +1,32 @@
+name: Pylint
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        #python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.9", "3.10"]
```

Comment:
@yuyiguo here you specify explicit versions of python which implies that when new version will come it will not be covered, and eventually this workflow may fail if underlying OS will not provide it. Do you want this? Why not to use `3.X` instead?

Reply:
@vkuznet 
I updated it to :
`python-version: ["3.x"]`
Please review it again.